pkg/utils/requester.go

```diff
@@ -76,18 +86,18 @@ func Headers(headers ...Header) func(r *HTTPRequester) {
 
 // HTTPRequester contains main info
 type HTTPRequester struct {
-	url     string
+	api     string
 	client  http.Client
 	retries int
 	headers []Header
 }
 
 // NewHTTPRequester makes Requester with api and parameters. Sets defaults
-// url has a complete url of the request like https://cdn.optimizely.com/datafiles/24234.json
-func NewHTTPRequester(url string, params ...func(*HTTPRequester)) *HTTPRequester {
+// api has the base part of request's url, like http://localhost/api/v1
+func NewHTTPRequester(params ...func(*HTTPRequester)) *HTTPRequester {
 
 	res := HTTPRequester{
-		url:     url,
+		api:     DatafileAPI,
```

Comment:
I don't think we should be specifying the APIs for the requester like this. The requester should remain agnostic of what the API or base URL is. We should let it just take any URL and send the http request to it.